sql/wsrep_trans_observer.h

```diff
@@ -540,6 +540,7 @@ static inline void wsrep_after_command_after_result(THD* thd)
   {
     thd->wsrep_cs().after_command_after_result();
   }
+  thd->wsrep_ctas= false;
```

Comment:
This happens too late. If the CTAS was executed as a part of stored procedure or multi statement query, the `wsrep_ctas` may cause side effects for all statements that follow CTAS.